client/app/scripts/superdesk-desks/aggregate.js

```diff
@@ -66,6 +69,44 @@
             setupCards();
         };
 
+        /**
+         * Return true if the 'fileType' filter is selected
+         * param {string} fileType
+         * @return boolean
+         */
+        this.hasFileType = function(fileType) {
+            return this.selectedFileType === fileType;
+        };
+
+        /**
+         * Set the current 'fileType' filter
+         * param {string} fileType
+         */
+        this.setFileType = function(fileType) {
```

Comment:
maybe we can put this filter into some `monitoring` service and make the groups watch that?
it would be simpler than updating each group. simpler to test (spoiler warning: there is no unittest for this method..)